src/runtime.js

```diff
@@ -42,13 +42,13 @@ class FetchEvent extends Event {
 
 window.FetchEvent = FetchEvent;
 
-export async function serve(addr) {
+export async function serve(addr = 'localhost:8080') {
   if (typeof addr === "string") {
     const [hostname, port] = addr.split(":");
     addr = { hostname, port: Number(port) };
   }
   const listener = Deno.listen(addr);
-  const host = `localhost:${listener.addr.port}`;
+  const host = `${listener.addr.hostname}:${listener.addr.port}`;
```

Comment:
We can assign a temporary variable here to hold the hostname to be displayed to the console.
```javascript
const outputHostname = addr.hostname || "localhost";
const listener = Deno.listen(addr);
const host = `${outputHostname}:${listener.addr.port}`;
```